inspirehep/modules/references/processors.py

```diff
@@ -90,11 +90,10 @@ def _is_arxiv(obj):
     """Small checker for arXiv report numbers."""
     # Workaround until this issue:
     # https://github.com/inveniosoftware/idutils/issues/14 is solved.
-
     arxiv_test = obj.split()
     if not arxiv_test:
         return False
-    return idutils.is_arxiv(arxiv_test[0])
+    return idutils.is_arxiv(arxiv_test[0]) or False
```

Comment:
because the match can return None. This means that the function returns a Match object if there is a match, None if there is not, and False if the split fails. 
Since the signature is `_is_arxiv` I would expect False if it is not an arxiv value, else the matched value.